README.md

```diff
@@ -1,15 +1,74 @@
 # Ionic Langchain
 
-Pre-release. Do not use this.
+Ionic Langchain provides a wrapper around the Ionic Commerce's SDK for use as a `Tool` in a custom Langchain agent.  This tool will enable e-commerce for your agent, allowing your users to ask for product recommendations and purchase products through the agent chat interface.
 
 ## Installation
 
-Pre-release. Do not use this.
+This tool requires at least `langchain@0.0.350` and can work with any greater patch release the `0.0.x` series.
+
+You can install the package from GitHub using `pip`:
+
+```sh
+pip install git+https://github.com/ioniccommerce/ionic_langchain.git#v0.1.2
+```
+
+or `poetry`:
+
+```sh
+poetry add git+https://github.com/ioniccommerce/ionic_langchain.git#v0.1.2
+```
 
 ## Usage
 
-Pre-release. Do not use this.
+```python
```

Comment:
more or less adapting what we have written in our demobot; happy to make modifications.